ui.frontend/src/utils.js

```diff
@@ -316,6 +317,22 @@ class Utils {
         }
     }
 
+    static async registerCustomFunctionsByUrl(url) {
+        if (url != null && url.trim().length > 0) {
+            const customFunctionModule = await import(/*webpackIgnore: true*/ url);
```

Comment:
comment would be needed for why have we added webpackIgnore

Reply:
done